docs/contributing-docs.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to the Docs
+
+Our documentation is written as Markdown files and served in GitHub pages
+by [MkDocs](https://www.mkdocs.org/).
+
+## Writing the docs
+
+As our documentation is written in Markdown, 
+the [base Markdown specification](https://daringfireball.net/projects/markdown/) 
+is a useful reference. MkDocs also includes 
+[some documentation to get you started writing in Markdown](https://www.mkdocs.org/user-guide/writing-your-docs/#writing-with-markdown).
+
+In addition to standard Markdown, our documentation supports the following extensions:
+
+- [Admonitions](https://python-markdown.github.io/extensions/admonition/)
+    adds specially called-out text anywhere within the document 
+    as notes, warnings, and other types.
+
+- [BetterEm](https://facelessuser.github.io/pymdown-extensions/extensions/betterem/) 
+    improves the handling of bold and italics.
+
+- [MagicLink](https://facelessuser.github.io/pymdown-extensions/extensions/magiclink/) 
+    provides automatic linking for URLs in the Markdown text.
+
+- [SuperFences](https://facelessuser.github.io/pymdown-extensions/extensions/superfences/)
+    makes a number of improvements to standard Markdown fences.
+
+- [Tilde](https://facelessuser.github.io/pymdown-extensions/extensions/tilde/) 
+    adds support for creating `<del></del>` tags with `~~`.
+
+- [Tables](https://python-markdown.github.io/extensions/tables/)
+    adds support for tables to standard Markdown.
+
+When creating new documents, they should be added to the 
+[mkdocs.yml file](https://github.com/cfpb/cfgov-refresh/blob/master/mkdocs.yml) 
+in the appropriate place under `nav:` with its table of contents title. 
+For example:
+
+```
+nav:
+- Introduction: index.md
+```
+
+## Running the docs locally
+
+### With Docker
+
+When running cfgov-refresh using [Docker-compose](https://cfpb.github.io/cfgov-refresh/installation/#docker-compose-installation)
+this documentation is running by default at http://localhost:8888.
+
+### Manually
+
+When using 
+[the stand-alone installation](http://localhost:8888/installation/#stand-alone-installation) 
+of cfgov-refresh, 
+you can run these docs with:
+
+```bash
+workon cfgov-refresh
```

Comment:
Should we suggest a docs-specific virtualenv so that we don't pollute the main one with docs requirements?

Reply:
I don't think so. @chosak had also suggested just adding `docs.txt` to `local.txt`. In that spirit I'm going to leave the `cfgov-refresh` venv here and we can change it if we need to.